src/client/app/App.js

```diff
@@ -1,7 +1,59 @@
-import React from 'react';
+import React, { Component } from 'react';
 
-const App = () => (
-  <h2>Stats Widget</h2>
-);
+import './app.css';
+
+class App extends Component {
+  static priceRanges(prices) {
```

Comment:
uhm, interesting. not a big fan of static methods to be honest.

Reply:
the linter notes to use static method when there is no `this` keyword in the class method which is my reason for doing this.